app/src/main/java/com/gathering/android/event/eventdetail/EventDetailScreen.kt

```diff
@@ -114,4 +179,171 @@ class EventDetailScreen : Fragment(), EventDetailNavigator {
             bundle
         )
     }
+
+    @Preview(showBackground = true)
+    @Composable
+    fun EventDetailPreview() {
+        EventDetail(
+            eventName = "Ani",
+            eventHostEmail = "animan@gmail.com",
+            description = "party",
+            photoUrl = "",
+            address = "101 Erskine Ave",
+            date = "15 Nov 2023",
+            time = "15:30",
+            acceptType = AcceptType.No,
+            {},
+            {},
+            {},
+            {}
+        )
+    }
+
+    @Composable
+    fun EventDetail(
+        eventName: String,
+        eventHostEmail: String,
+        description: String,
+        photoUrl: String,
+        address: String,
+        date: String,
+        time: String,
+        acceptType: AcceptType,
+        onYesButtonClick: () -> Unit,
+        onNoButtonClick: () -> Unit,
+        onMaybeButtonClick: () -> Unit,
+        onAttachListClicked: () -> Unit,
+    ) {
+        Column(
+            Modifier
+                .padding(10.dp)
+                .fillMaxSize()
+        ) {
+            EventImage(photoUrl)
+            Text(
+                fontSize = 20.sp,
+                fontWeight = FontWeight.Bold,
+                text = eventName,
+                modifier = Modifier.padding(start = 10.dp, 40.dp)
+            )
+            Text(
+                fontSize = 16.sp,
+                text = eventHostEmail,
+                modifier = Modifier.padding(10.dp)
+            )
+            Text(
+                fontSize = 16.sp,
+                text = description,
+                modifier = Modifier.padding(10.dp)
+            )
+            Text(
+                fontSize = 16.sp,
+                text = address,
+                modifier = Modifier.padding(10.dp)
+            )
+            Text(
+                fontSize = 16.sp,
+                text = date,
+                modifier = Modifier.padding(10.dp)
+            )
+            Text(
+                fontSize = 16.sp,
+                text = time,
+                modifier = Modifier.padding(10.dp)
+            )
+            Button(
```

Comment:
i think it is better to create a composable for this button like this:

```
@Composable
fun AttendeeListButton(onClick: () -> Unit) {
    Button(
        colors = ButtonDefaults.buttonColors(
            containerColor = Color.Transparent,
            contentColor = Color.DarkGray
        ),
        onClick = onClick,
        content = {
            Text(
                fontWeight = FontWeight.Bold,
                fontSize = 20.sp,
                text = "Click to see attendee",
            )
        }
    )
}
```

then calling it in the EventDetail like so:

AttendeeListButton(onAttachListClicked)